dagql/idtui/frontend_pretty.go

```diff
@@ -687,8 +688,10 @@ func (fe *frontendPretty) update(msg tea.Msg) (*frontendPretty, tea.Cmd) { //nol
 		case "home":
 			fe.goStart()
 			return fe, nil
-		case "end", " ":
+		case "end", "G", " ":
 			fe.goEnd()
+			fe.pressedKey = "end"
```

Comment:
The alternative to faking these out is to list `G` in the keys in the keymap, but it's not like that's much better since it's duplicative.

Someday maybe we should probably refactor into a data structure that drives both the UI and the key interpretation.